chord_metadata_service/chord/views_data_types.py

```diff
@@ -0,0 +1,140 @@
+from bento_lib.responses import errors
+from django.core.exceptions import ValidationError
+from django.db.models import QuerySet
+from django.http import HttpRequest
+
+from adrf.decorators import api_view
+from rest_framework import status
+from rest_framework.decorators import permission_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+from typing import Optional
+
+from chord_metadata_service.phenopackets.models import Phenopacket
+from chord_metadata_service.experiments.models import Experiment, ExperimentResult
+
+from . import data_types as dt
+
+OUTPUT_FORMAT_VALUES_LIST = "values_list"
+OUTPUT_FORMAT_BENTO_SEARCH_RESULT = "bento_search_result"
```

Comment:
Just noticed OUTPUT_FORMAT_VALUES_LIST and OUTPUT_FORMAT_BENTO_SEARCH_RESULT are defined but not used anywhere. can they be removed for now?

Reply:
oh yeah theyre a copy paste mistake from views_search, thanks!